### PR TITLE
Alpha release touchups

### LINKED ISSUE #111: change banner to alpha from pre-alpha

### PR DESCRIPTION
**Pull request by lboeman**
closes #111 
:clap: :champagne:

### CODE REVIEW

**Comment by lboeman** [comment]
added units to creation forms to close #65

File: sfa_dash/templates/banner.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="container">
-  <p>This is a pre-Alpha release of the Solar Forecast Arbiter dashboard. If you encounter any issues please send us a report using our
+  <p>This is the Alpha release of the Solar Forecast Arbiter dashboard. If you encounter any issues please send us a report using our
```
Comment (wholmgren) [inline sfa_dash/templates/banner.html]: can we use versioneer to make this automatic? I imagine we'll have multiple alpha releases before the next milestone-mandated beta release.

Comment (lboeman) [reply sfa_dash/templates/banner.html]: Sure can!

**Comment by lboeman** [comment]
Added version printing to banner like so:
Screenshot from 2019-06-28 15-01-32

**Comment by alorenzo175** [comment]
Maybe add the release to the sentry_sdk init in ``sfa_dash/__init__.py``.
Something like 
```
release=f'sfa-dash@{__version__}'
```

**Pull request merged**